apps/gk_species_radiation.c

```diff
@@ -79,6 +78,16 @@ gk_species_radiation_init(struct gkyl_gyrokinetic_app *app, struct gk_species *s
     }
 
     int status = gkyl_get_fit_params(*rad_data, s->info.radiation.z[i], s->info.radiation.charge_state[i], a, alpha, beta, gamma, v0, s->info.radiation.num_of_densities[i]);
+    double *vtsq = gkyl_array_fetch(rad->vtsq_min, i);
+    double T_min_eV;
```

Comment:
Because this conditional block includes important comments, let's do syntax like:

if {
// comment
T_min_ev = ...
}
else if {
// comment
T_min_ev = ...
} 

etc.